rdata/unparser/_ascii.py

```diff
@@ -68,16 +68,50 @@ def _unparse_array_values(self, array: npt.NDArray[Any]) -> None:
 
             self._add_line(line)
 
-    def unparse_string(self, value: bytes) -> None:
-        """Unparse a string."""
-        self.unparse_int(len(value))
-
+    def _unparse_string_characters(self, value: bytes) -> None:
```

Comment:
I wonder if it wouldn't be easier, faster and more elegant to just create a global list of length 256 and use that for the mapping, instead of doing all this logic for each string.

Reply:
Good idea! I did that change in 918ea23.